src/clj/harja/palvelin/raportointi/raportit/vastaanottotarkastus.clj

```diff
@@ -263,49 +294,137 @@
+                            :pk2-pituus (+ (:pk2-pituus yht-rivi) (:pk2-pituus rivi))
+                            :pk3-pituus-yotyo (+ (:pk3-pituus-yotyo yht-rivi) (:pk3-pituus-yotyo rivi))
+                            :pk3-pituus (+ (:pk3-pituus yht-rivi) (:pk3-pituus rivi))
+                            :eitiedossa-pituus-yotyo (+ (:eitiedossa-pituus-yotyo yht-rivi) (:eitiedossa-pituus-yotyo rivi))
+                            :eitiedossa-pituus (+ (:eitiedossa-pituus yht-rivi) (:eitiedossa-pituus rivi))))
+        valtakunnallisesti-yhteensa (reduce
+                                      fn-laske-yhteen
+                                      (yhtrivi-pohja "Valtakun. yht. / KA%")
+                                      urakkarivit)
+
+        ;; Groupataan ja järjestellään rivit hallintayksiköittäin, kuten pk-luokka taulukossakin
+        elyttain-jaoteltu (sort-by first (group-by :hallintayksikko_nimi urakkarivit))
+
+        formatoi-elyt-fn (fn [ely]
+                           (let [elyrivi {:otsikko (str (:elynumero (first (second ely))) " " (first ely))}
+                                 urakat-lista (mapv
+                                                (fn [rivi]
+                                                  (pkluokka-yotyo-rivi rivi false false))
+                                                (second ely))
+                                 ely-yhteensa (reduce fn-laske-yhteen
+                                                (yhtrivi-pohja (str (first ely) " yht. / KA%"))
+                                                (second ely))]
+
+                             (vec (flatten [elyrivi (into [] urakat-lista) (pkluokka-yotyo-rivi ely-yhteensa true true)]))))
+
+        valtukunnallinen-rivi (pkluokka-yotyo-rivi valtakunnallisesti-yhteensa true true)]
+
+    [:taulukko {:tyhja (when (empty? urakkarivit) "Ei kohteita.")
+                :sheet-nimi "Yötyö / PK-luokka"
+                :rivi-ennen [{:sarakkeita 1 :leveys 10}
+                             {:teksti "PK1 yötyö"
+                              :sarakkeita 2
+                              :luokka "paallystys-tausta-tumma"
+                              :tummenna-teksti? true
+                              :tasaa :keskita}
+                             {:teksti "PK2 yötyö"
+                              :sarakkeita 2
+                              ;;:luokka "paallystys-tausta-tumma"
+                              :tummenna-teksti? true
+                              :tasaa :keskita}
+                             {:teksti "PK3 yötyö"
+                              :sarakkeita 2
+                              :luokka "paallystys-tausta-tumma"
+                              :tummenna-teksti? true
+                              :tasaa :keskita}
+                             {:teksti "Ei tiedossa yötyö"
+                              :sarakkeita 2
+                              ;;:luokka "paallystys-tausta-tumma"
```

Comment:
Kommentit voi heittää mäkeen, näitä ei ilmeisesti tarvita

Reply:
Joo. Ei noissa kohdissa. Mutta lisätty niihin oikeisiin kohtiin.